packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -151,6 +72,96 @@ class TimelineBodyState extends State<TimelineBody> {
     );
   }
 
+  Widget _buildLeftAlignedButtons() {
```

Comment:
nit: _buildLeftAlignedButtons is a bit clumsy. How about a name that covers what sorts of UI actions these buttons do and tweak the return type to be a List<Widget> leaving the styling of the content to the outer build method?
I can't think of a good name but maybe: recordingSettingButtons.

Reply:
Renamed `_buildLeftAlignedButtons` to `_buildTimelineStatButtons`.
Renamed `_buildRightAlignedButtons` to `_buildSecondaryButtons`. The right group of buttons don't really share a common theme - a button that modifies a VM flag, a button that calls a service extension, and a button that exports the timeline. Pulling each button out by its function seems a bit overkill and would defeat the purpose of grouping them at all for readability.